internal/presto/columns.go

```diff
@@ -769,7 +769,7 @@ func (b *PrestoThriftTimestamp) At(index int) interface{} {
 		return nil
 	}
 
-	return b.Timestamps[index]
+	return time.Unix(b.Timestamps[index]/1000, 0)
```

Comment:
Oh I see. Hmm this error seems quite fundamental and Talaria Processor should have surfaced it awhile ago, I thought we did a detailed data comparison 🤔. Perhaps it is interpreted and converted by Spark. @a9kitkumarsinha